src/test/kotlin/lotto/LottoTest.kt

```diff
@@ -0,0 +1,23 @@
+package lotto
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class LottoTest {
```

Comment:
```suggestion
class LottoGeneratorTest {
```
구현부를 보니, LottoTest보다는 LottoGeneratorTest에 가까워보입니다. 🙏

Reply:
LottoGeneratorTest로 수정하였습니다!